### DISCUSSION TITLE
BPMN graphic painter for ViewFlow?

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [viewflow/viewflow#405] [General]
Is there a graphic BPMN Workflow Painter that is compatible with ViewFlow?

**Comment by michaeloliveraz** [comment]
the reason I am looking for this is our customers and users will NOT write code, do not want to pay developers to write code, and want the ability to graphically create,, edit, and delete workflows.

**Comment by kmmbvnr** [comment]
Viewflow is the thin workflow layer on top of Django web framework allowing to wirte workflows in code, and utilize the standard dev practices like unit-test the code, using git as the version history, and standard test-staging-prod pipeline
